py/misc/compress.py

```diff
@@ -0,0 +1,40 @@
+import sys
+import os
+import csv
+from zipfile import ZipFile
+import time
+import logging
+import logging.handlers as handlers
+
+
+def compress_csv():
+    """Compress downloaded .csv files"""
+    if not os.path.exists(csv_path):
+        os.makedirs(csv_path)
+    os.chdir(csv_path)
+    try:
+        zip_csv = ZipFile(site_name + '_' + date + '_csv.zip', 'a')
+        for file in glob.glob("*" + date + "*" + "csv"):
+            zip_csv.write(file)
+            os.remove(file)
+        logging.info("Compressing %s item(s)", str(OBSERVATION))
+    except Exception as e:
+        logging.error('Error when compressing csv')
+        logging.info(type(e).__name__ + str(e))
+    os.chdir(project_path)
```

Comment:
We are not going to use the global value `OBSERVATION` anymore. Lets make it one of the function's argument.

See: https://docs.quantifiedcode.com/python-anti-patterns/maintainability/using_the_global_statement.html

Reply:
Also `csv_path`, `date`, `site_name` and `project_path`. Change everything undefined in this module in functions' parameters.